### DISCUSSION TITLE
Does it support Remote Access?

### DISCUSSION THREAD

**Discussion by kishan-dhankecha** [mistweaverco/bananas#28] [Q&A] [answered]
Am I able to control the mouse cursor of the remote desktop and also write to the remote dashboard?

**Comment by gorillamoe** [comment, marked as answer]
Not yet, but planned for v1.

As of now, the participant can just guide the host by having it's own cursor drawn onto the hosts screen and ping with double click to mark an important section of the screen.

Full takeover of mouse and keyboard is planned, but needs some real thinking going into it, because that's really important from a security point of view, that e.g. there is a global shortcut to kill remote control immediately.

The benefit of using bananas as of now is the crystal clear transmission of the video and also having a way to guide the host 

Imagine a normal pairing session via Google meet:

See the code here at line 22, no, more to the left, no right there, no at the getCursorAtX .. no.. yes there .

Or when showing in a UI it even gets worse.. click near the header, no more to the right, no, the other header, almost, more to the right, yes.. here.

With a remote cursor and a ping, it's instantly visible what you mean.

**Reply by gorillamoe** [reply]
You can have a look at the roadmap here: https://github.com/mistweaverco/bananas/milestones
